Furyfile.yml

```diff
@@ -10,8 +9,45 @@ roles:
 modules:
   - name: aws/aws-vpc
   - name: aws/aws-kubernetes
+  - name: terraform-aws-acm
+    version: v2.5.0
+    provider:
+      name: aws
+      label: ufficial-modules
```

Comment:
Please, replace all `ufficial` to `official`

Reply:
italianglish prevails over english, my typo